python/bolt12.py

```
@@ -111,24 +117,174 @@ def simple_bech32_decode(bech32str: str) -> Tuple[str, bytes]:
         return None, None
 
     ret5 = bytes()
-    for c in bech32str[sep+1:]:
+    for c in bech32str[sep + 1:]:
         pos = bech32.CHARSET.find(c)
         if pos == -1:
             return None, None
```

Here as well, it should throw an error with the invalid-letter?